### PR TITLE
Sagemaker DataWrangler Samples addition

### PR DESCRIPTION
**Pull request by neelamkoshiya**
*Issue #, if available:*

*Description of changes:*

*Testing done:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [ X] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [X ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [ NOT APPLICABLE] I have tested my notebook(s) and ensured it runs end-to-end
- [ NOT APPLICABLE] I have linted my notebook(s) and code using `tox -e black-format,black-nb-format`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. 

(Details about this : https://t.corp.amazon.com/P68066859)

### CODE REVIEW

**Review by aaronmarkham — DISMISSED** [review]

File: sagemaker-datawrangler/joined-dataflow/explore_data.ipynb [hunk 1]
```diff
@@ -0,0 +1,281 @@
+{
```
Comment (atqy) [inline sagemaker-datawrangler/joined-dataflow/explore_data.ipynb:1]: <ol><li>Please include a title with H1 heading.</li><li>Writer an introductory passage explaining what this notebook is doing</li><li>Write some descriptions for what you are doing in the cell (i.e. import the necessary packages, etc.).</li><li>In the case that this notebook is part of a sequence of notebooks (i.e. a set of notebooks that should be run in a sequence), please refer to them as well and make it clear in the intro that the notebook is part of a sequence so that the audience is aware. Take a look at the introduction of https://github.com/aws/amazon-sagemaker-examples/blob/main/end_to_end/music_recommendation/01_music_dataprep.flow for an example</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (atqy) [inline sagemaker-datawrangler/joined-dataflow/explore_data.ipynb:1]: <p>Write some descriptions for what you are doing in the cell (i.e. import the necessary packages, etc.).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (atqy) [inline sagemaker-datawrangler/joined-dataflow/explore_data.ipynb:1]: <p>Write some descriptions for what you are doing in the cell (i.e. import the necessary packages, etc.).</p><p> </p><p>Also, maybe it is best to move this to the "Prerequisites: Get Data" section since it is used there?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (atqy) [inline sagemaker-datawrangler/joined-dataflow/explore_data.ipynb:1]: <p>Missing descriptions of how you are exploring the data and what you are exploring exactly. The audience may be confused at what you are trying to do here, especially for beginners</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (atqy) [inline sagemaker-datawrangler/joined-dataflow/explore_data.ipynb:1]: <p>Is this supposed to be here? Is this notebook part of a series of sequential notebooks? If not, please remove this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

File: sagemaker-datawrangler/joined-dataflow/explore_data.ipynb [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
```
Comment (atqy) [inline sagemaker-datawrangler/joined-dataflow/explore_data.ipynb:1]: <ul><li>Use present tense as per guidelines (i.e. we download and explore the data)</li><li>I would reword "In this notebook, we will download and explore the data that is used to build the SageMaker DataWrangler flow file for data processing." as "In this notebook, we download and explore the data. To prepare it for a machine learning task, we then process this data by writing a SageMaker DataWrangler .flow file."</li><li>I would reword "After running this notebook, you can follow the <a href="https://github.com/aws/amazon-sagemaker-examples/raw/ecf4ab542d371fe5131106285c27379971875497/sagemaker-datawrangler/joined-dataflow/README.md" target="_blank">README.md</a> for the step by step instructions to build the data processing flow file to prepare data for the machine learning task." to "After running this notebook, you can follow the <a href="https://github.com/aws/amazon-sagemaker-examples/raw/ecf4ab542d371fe5131106285c27379971875497/sagemaker-datawrangler/joined-dataflow/README.md" target="_blank">README.md</a> for step by step instructions how to write the SageMaker DataWrangler .flow file."</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (atqy) [inline sagemaker-datawrangler/joined-dataflow/explore_data.ipynb:1]: <p>Use present tense ("Here we download the music dataset from a public S3 bucket")</p><p> </p><p>I would reword the whole thing as "Here, we download the music data from a public S3 bucket. We then upload it to your default S3 bucket, which was created for you when you initially created a SageMaker Studio workspace. "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (atqy) [inline sagemaker-datawrangler/joined-dataflow/explore_data.ipynb:1]: <p>Use present tense "we perform preliminary data exploration to understand the data"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (atqy) [inline sagemaker-datawrangler/joined-dataflow/explore_data.ipynb:1]: <p>Use present tense -&gt; "After completing this notebook, you can follow the steps in the README to start building the DataWrangler flow file."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

File: sagemaker-datawrangler/import-flow.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+## Import Flow
+
+Each of the example has a flow file available which you can directly import to expediate the process or validate the flow.
```
Comment (atqy) [inline sagemaker-datawrangler/import-flow.md:3]: Spelling mistake: should be expedite

File: sagemaker-datawrangler/joined-dataflow/index.rst [hunk 4]
```diff
@@ -0,0 +1,701 @@
+
+Joined Dataflow Example
+============================
+This example provide quick walkthrough of how to aggregate and prepare
```
Comment (atqy) [inline sagemaker-datawrangler/joined-dataflow/index.rst:4]: Grammer: "the example provides"

File: sagemaker-datawrangler/tabular-dataflow/data-transformations/index.rst [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+
+.. figure:: https://s3.amazonaws.com/sagemaker-sample-files/images/sagemaker-datawrangler/tabular-dataflow/drop-more-cols.png
+
+If the Preview is OK, click **Add** to add the transform in the data
+flow.
+
+Drop Duplicate Rows
+-------------------
+
+To drop the duplicate rows that we identified based on the analysis we
```
Comment (atqy) [inline sagemaker-datawrangler/tabular-dataflow/data-transformations/index.rst:73]: "To drop the duplicate rows" - > "We should drop the duplicate rows"

File: sagemaker-datawrangler/tabular-dataflow/index.rst [hunk 6]
```diff
@@ -0,0 +1,361 @@
...
+Dataset
+~~~~~~~
+
+.. raw:: html
+
+   <div class="alert alert-block alert-info">
+
+Dataset
+
+We will be using the `Hotel Booking Demand
```
Comment (atqy) [inline sagemaker-datawrangler/tabular-dataflow/index.rst:30]: Use present tense - "We use the "

**Review by atqy — APPROVED** [review]
